portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -125,8 +132,12 @@ $.extend(true, doaj, {
                         </div>\
                     </header>';
                 }
-
-                var frag = titleBarFrag + '\
+                var frag = "";
```

Comment:
I'm not entirely sure why github shows that as a difference. The publicSearch template has not been shown, `if(this.resultsOnly)` has been here since the beginning. The only difference to this file should be FQWidget template added.

Reply:
It looks like it's just because of the `resultsOnly` if statement that's been inserted.  it's all fine.